pyproject.toml

```diff
@@ -49,6 +50,9 @@ nbdime = "^2.0.0"
 jupytext = "^1.4.2"
 ipywidgets = "^7.5.1"
 pytest-sugar = "^0.9.3"
+snakeviz = "^2.1.0"
+jupyter_http_over_ws = "^0.0.8"
+jupyterlab_github = "^2.0.0"
```

Comment:
Are you intentionally adding this dev dependency?

Reply:
I can remove them. What is the rational for what should be a dev dependency?